tests/test_common.py

```diff
@@ -355,11 +358,18 @@ def __init__(self, *args, **kwargs):
             '--log_file', TestCommon.log_file
         ]
         os.environ['CLARITYSCRIPTCONFIG'] = join(self.etc_path, 'example_clarity_script.yml')
+        self.current_wd = os.curdir
```

Comment:
This does not give the path to the working dir, but is an alias for whatever the operating system uses to refer to the current directory, which usually is the string `.`. Were we meaning to do `os.getcwd()`?

Reply:
Yes it should be changed to `os.getcwd()`

Reply:
Updated